.github/workflows/samples_configuration.yml

```diff
@@ -16,7 +16,7 @@ env:
   IS_IN_CI_PIPELINE: "true"
 
 jobs:
-  samples_notebook_ci:
+  samples_configuration:
```

Comment:
maybe we can make this a separate PR to do so?

Reply:
I use this part for testing, changing is OK.